src/images/vector.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="8" height="10" viewBox="0 0 8 10" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
jest już przez Adriana zrobione icons.svg, ale to potem sie przerzuci, żeby wszystkie ikony były gdzieś razem